RiotSwiftUI/Modules/AnalyticsPrompt/AnalyticsPromptViewModel.swift

```diff
@@ -19,7 +19,7 @@ import Combine
 
 @available(iOS 14, *)
 typealias AnalyticsPromptViewModelType = StateStoreViewModel<AnalyticsPromptViewState,
-                                                             AnalyticsPromptStateAction,
+                                                             Never,
```

Comment:
I think we can just get rid of `StateAction` Generic altogether?

Reply:
I figured I'll leave it in so that Gil doesn't have problems with his Spaces screens after merging develop. 
I'll remove it entirely once we migrate those screens.